DeepCat.js

```diff
@@ -562,14 +565,32 @@
 	}
 
 	function addSmallFormHint() {
-		var smallHintBox = '<div id="deepcat-smallhint">'
+		var smallHintBox = '<div id="deepcat-smallhint" style="display: none;">'
 			+ '<img id="deepcat-smallhint-hide" title="' + mw.msg( 'deepcat-smallhint-close' ) + '" src="https://upload.wikimedia.org/wikipedia/commons/4/44/Curation_bar_icon_close.png">'
 			+ mw.msg( 'deepcat-hintbox-small' )
 			+ '</div>';
 		$( '#searchform' ).after( smallHintBox );
 		$( '#deepcat-smallhint-hide' ).on( 'click', hideSmallHint );
 	}
 
+	function initializeHintCodeIfNeeded() {
+		if( hintCodeLoaded ) {
+			return;
+		}
+		mw.loader.load( cssPath + 'DeepCat.hintbox.css&action=raw&ctype=text/css', 'text/css' );
```

Comment:
Don't we want to put those CSS paths in some config-like variable (on top of the file or so), so it is easier to customize and paths are not hidden somewhere deep in private methods?